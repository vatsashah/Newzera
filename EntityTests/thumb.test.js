// Importing required library
import React from 'react'
import { shallow } from 'enzyme'

// Importing component to be tested
import Thumb from '../../../Components/Entity/thumb'

describe('Thumbnails should render without any error', () => {
  it('Thumbnails should render correctly', () => {
    const component = shallow(<Thumb />)
    expect(component).toMatchSnapshot()
  })
})
