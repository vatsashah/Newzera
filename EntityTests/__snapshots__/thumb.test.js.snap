// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbnails should render without any error Thumbnails should render correctly 1`] = `
<div
  className="ThumbContainer"
>
  <div
    key="0"
  >
    <img
      alt="Thumb"
      className="ThumbImage"
      src="ThumbOne.png"
      style={
        Object {
          "left": "0vw",
          "top": "0vh",
        }
      }
    />
  </div>
  <div
    key="1"
  >
    <img
      alt="Thumb"
      className="ThumbImage"
      src="ThumbTwo.png"
      style={
        Object {
          "left": "1.8229166666666667vw",
          "top": "0vh",
        }
      }
    />
  </div>
  <div
    key="2"
  >
    <img
      alt="Thumb"
      className="ThumbImage"
      src="ThumbThree.png"
      style={
        Object {
          "left": "3.6458333333333335vw",
          "top": "0vh",
        }
      }
    />
  </div>
  <div
    key="3"
  >
    <img
      alt="Thumb"
      className="ThumbImage"
      src="ThumbFour.png"
      style={
        Object {
          "left": "5.46875vw",
          "top": "0vh",
        }
      }
    />
  </div>
</div>
`;
